lib/cocoapods/installer/analyzer.rb

```diff
@@ -865,7 +865,7 @@ def determine_platform(specs, target_definitions, build_type)
           Version.new(library_spec.deployment_target(platform_name) || default)
         end.max
         if platform_name == :ios && build_type.framework?
-          minimum = Version.new('8.0')
+          minimum = Version.new('11.0')
```

Comment:
This is probably the wrong place in the code for this change, since this is specifying 8.0 is when Apple introduced framework support for iOS. The change should also impact tvOS, macOS, and tvOS

Reply:
oh now I understood

Reply:
@paulb777 
how does it possible to impact other OS(tvOS, macOS and tvOS) the change only run when the `platform_name` is `ios`because of the top line code 
`if platform_name == :ios && build_type.framework?`

Reply:
and I think it right place to change cause I change every singe code that determine the version `8.0` to `12.0` one by one and only place that can help was the `lib/cocoapods/installer/analyzer.rb`

Reply:
macos and tvos have the same issue as ios and should have a similar fix.

Reply:
ah now I see, but currently this fix is for iOS, so what about fix other platform's issue on the other commit? (I cannot find the source for other platform on this file)

Reply:
As described, this is the wrong place for managing minimum version. This code is for testing if frameworks are supported or not on iOS.